unit_test.sh

```diff
@@ -16,6 +16,11 @@ DATABASE_USER="compliance"
 DATABASE_PASSWORD="changeme"
 DATABASE_NAME="compliance-test"
 
+# if this is a PR, use a different tag, since PR tags expire
```

Comment:
Is this something to make it more convenient for us or what does it fix?

Reply:
Just copied and pasted what I was told to copy and paste: https://github.com/RedHatInsights/bonfire/blob/master/cicd/bootstrap.sh#L23

Reply:
Okay. What it is good for?

Reply:
The expiry date on Images tags has been there since always, the only thing that has changed is the format (for whatever reason). 

So ... before we had this 7 characters long image tag from the commit's HASH, and if it was created on a PR, the tag had the `expiry-date` metadata added to it so the PR image tags didn't stay forever in Quay. 

For some reason, someone maintaining the shared "build.sh" script from `bonfire` project changed the format too, being this `pr-1058-52cf345` as an example. So this change is just affecting us because we use the shared `build.sh` script from Bonfire repo ([this one](https://github.com/RedHatInsights/bonfire/blob/master/cicd/build.sh))

Reply:
Thanks @Victoremepunto for explanation. Those are perks of depending on a dynamic code from main branch.